client/components/EyesWanted/AddEyesWanted.vue

```diff
@@ -99,7 +130,8 @@
         } catch (e) {
```

Comment:
Also would be good to fix copy paste error on line 127 to change message to be about eyes wanted instead of thanks (i cant comment on that section so its here oops)